fractal/cmd/_task.py

```diff
@@ -110,16 +110,24 @@ async def post_task(
     source: str,
     input_type: str = "Any",
     output_type: str = "Any",
+    batch: bool = False,
     version: Optional[str] = None,
     meta_file: Optional[str] = None,
-    batch: bool = False,
+    args_schema: Optional[str] = None,
+    args_schema_version: Optional[str] = None,
 ) -> BaseInterface:
     optionals = {}
     if version:
         optionals["version"] = version
     if meta_file:
         with open(meta_file, "r") as f:
             optionals["meta"] = json.load(f)
+    if args_schema:
+        with open(args_schema, "r") as f:
+            optionals["args_schema"] = json.load(f)
+        if args_schema_version:
```

Comment:
args_schema and args_schema_version are independent, and a user should be able to update only one of them